docker/docker-compose.cast.yml

```diff
@@ -11,6 +11,8 @@ services:
       DicomWeb__endpoint: "http://dicomserver:8080"
       DicomCastWorker__PollInterval: "00:00:05"
       Logging__Console__IncludeScopes: "true"
+      PatientSystemId: "patientId"
```

Comment:
Is `"patientid"` a reasonable default value?

Reply:
Updated to `patientSystemId`

Reply:
Is any custom string a reasonable value? Surely blank is the proper default, right? Or the use of an issuer tag?

Reply:
My understanding is that docker-compose file is for testing purpose in local. If that is the case, then we should keep it as a hardcoded value instead of blank. Please correct me if I am wrong.